### DISCUSSION TITLE
Modify the width of a fluent-option inside a listbox

### DISCUSSION THREAD

**Discussion by JeremyJ78** [microsoft/fluentui-blazor#3082] [General]
Hi All,

I'm in a worry...
I'm trying to change the width of a fluent-option...

![image](https://github.com/user-attachments/assets/10de55da-e2b1-441a-836d-6e93367e4b7f)

Here, on the image above, we can see that the date isn't at the right of the fluent-option...
It seems that the issue is the content part of the fluent-option...

![image](https://github.com/user-attachments/assets/724f3eee-e085-4d2c-96d1-f05d5dee3567)

Because if I change this directly in the debugguer : 

![image](https://github.com/user-attachments/assets/2d66530e-f042-4647-9877-68920057b01f)

The content inside the fluent-option is at the good place...

I tried many things to change the content part in the fluent-option but no way work :(

I tried such things :

`::deep fluent-option::part(content) {
width: 100%;
}`

`::deep fluent-option > content {
width: 100%
}`

`::deep fluent-option .content {
width: 100%
}`

And this is very strange because if we try to do the same thing for a fluent-tabs, if we wrote this : 

`::deep fluent-tabs::part(activeIndicator) {
width:90%;
}`

this work...

How can we change the width of the content part of the fluent-option ?

Thanks for help !
Best regards,

**Comment by dvoituron** [comment]
I think your first solution is good. Probably the `::deep` which generate an invalid result. 

Can you try using this style in your global `app.css` to avoid to use this `deep`?

```
fluent-option::part(content) { width: 100%; }
```

**Reply by JeremyJ78** [reply]
Really strange...
You are right... 
 Moving the rule to app.css made the rule work
Thank you:) It's not something I would have thought of

**Reply by oneolddev** [reply]
@JeremyJ78 

Can you mark this as answered. 😄
